### ISSUE TITLE
Make the warning "No C extensions available" an info-level log entry

### ISSUE THREAD

**Issue by lyw07** [learningequality/kolibri#4532]
### Observed behavior

The warning "No C Extensions available" when running kolibri makes users confused. It would be better if we hide it from most end-users

Please see the discussion here: https://learningequality.slack.com/archives/CB37UM23A/p1542131204155300

### Expected behavior

Users should not see the warning. It should be an `info` level event instead

### User-facing consequences

Users are confused about this warning.

### Errors and logs

```
<WARNING:root>:No C Extensions available for this platform.
```

### Steps to reproduce

Start kolibri in platforms such as Mac OS.

**Comment by Tlazypanda** [comment]
@indirectlylit can I take this up? Also can you guide me as to what should be the error message (couldn't find anything in the docs)? Thanks :v:

**Comment by indirectlylit** [comment]
let's hold off on this for a couple days while we figure out the best behavior

**Comment by indirectlylit** [comment]
Hi - I've updated this issue to reflect a decision that the message should be turned into an `info`-level log entry

**Comment by rtibbles** [comment]
Fixed by #7316

**Comment by indirectlylit** [comment]
thanks you!

Minor note: remember to set the appropriate milestone. Technically, the 'upcoming' milestones should never have any closed issues except for invalid ones.
